config/routes.rb

```diff
@@ -8,6 +8,8 @@
   }
   get 'up' => 'rails/health#show', as: :rails_health_check
 
-  resources :users, only: :show
+  resources :users, only: :show do
+    resources :tweets, only: %i[new create]
+  end
```

Comment:
This generates the following routes `/users/:user_id/tweets` and `/users/:user_id/tweets/new`. Since you are using the Devise helper `current_user` I believe there's no need to scope tweets to the `user_id`